understanding/22/focus-not-obscured-minimum.html

```diff
@@ -17,66 +17,66 @@ <h2>Status</h2>
+         <p>Another form of obscuring can occur where light boxes or other semi-opaque effects overlap the item with focus. While less than 100 percent opacity is not causing the component to be <q>entirely hidden</q>, such semi-opaque overlaps may cause a failure of <a href="../understanding/non-text-contrast.html">1.4.11 Non-text Contrast</a>. When a focus indicator can be covered by a semi-opaque component, the ability of the focus indicator to pass 1.4.11 should be evaluated (and pass) while the focus indicator is under the semi-opaque component. The intention in both situations is that the component receiving focus should never be obscured to the point a user cannot tell which item has focus.</p>
 
     </section>
 
-    <section id="benefits">
-      <h2>Benefits of Focus Not Obscured</h2>
+<section id="benefits">
+      <h2>Benefits of Focus Not Obscured (Minimum)</h2>
       
       <ul>
-         <li>Sighted users who rely on a keyboard interface to operate the page will be able to see the component which gets keyboard focus. Such users include those who rely on devices which use the keyboard interface, including speech input, sip-and-puff software, onscreen keyboards, scanning software, and a variety of assistive technologies and alternate keyboards.</li>
-         <li>People with limited or low vision, who may primarily user a pointer for screen orientation and repositioning, nonetheless benefit from a visible indication of the current point of keyboard interaction, especially where magnification reduces the overall viewing portion of the screen.</li>
-         <li>People with attention limitations, short term memory limitations, or limitations in executive processes benefit by being able to discover where the focus is located.</li>
+         <li>Sighted users who rely on a keyboard interface to operate the page will be able to see the component which gets keyboard focus. Such users include those who rely on devices which use the keyboard interface, including speech input, sip-and-puff software, on-screen keyboards, scanning software, and a variety of assistive technologies and alternate keyboards.</li>
+        <li>People with limited or low vision but who rely upon a pointing device (for viewport orientation and repositioning) from a visible indication of the current point of keyboard interaction, especially where magnification reduces the overall useable portion of content.</li>
```

Comment:
missing word + correction to american spelling convention:

```suggestion
        <li>People with limited or low vision but who rely upon a pointing device (for viewport orientation and repositioning) benefit from a visible indication of the current point of keyboard interaction, especially where magnification reduces the overall usable portion of content.</li>
```